Maskbook/Extension/URL.swift

```diff
@@ -15,3 +15,14 @@ extension URL {
             .urls(for: .documentDirectory, in: .userDomainMask)[0]
     }
 }
+
+extension URL {
+    public var queryParameters: [String: String]? {
+        guard
+            let components = URLComponents(url: self, resolvingAgainstBaseURL: true),
+            let queryItems = components.queryItems else { return nil }
+        return queryItems.reduce(into: [String: String]()) { (result, item) in
```

Comment:
Here we can use `[:]` instead of `[String: String]()`, as the return type can be inferred by the compiler.